src/annotator/selection-observer.ts

```diff
@@ -22,15 +19,21 @@ export function selectedRange(document) {
  * An observer that watches for and buffers changes to the document's current selection.
  */
 export class SelectionObserver {
+  private _pendingCallback: number | null;
```

Comment:
Perhaps add a comment on this field: It's not obvious without reading code that it is used to hold a timeout ID. (I know there wasn't a comment before, either).